pkg/addons/assets/vpc-admission-webhook-dep.yaml

```diff
@@ -33,9 +33,28 @@ spec:
               mountPath: /etc/webhook/certs
               readOnly: true
       hostNetwork: true
-      nodeSelector:
-        beta.kubernetes.io/os: linux
-        beta.kubernetes.io/arch: amd64
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: beta.kubernetes.io/os
+                operator: In
+                values:
+                - linux
+              - key: beta.kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+            - matchExpressions:
+              - key: kubernetes.io/os
+                operator: In
+                values:
+                - linux
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
```

Comment:
unlike coredns,kubeproxy and aws node we don't offer the user to update the deployment. In this case I think having both labels supplied makes sense, it prevents any users experiencing breaking changes during an upgrade.